js/index.js

```diff
@@ -16,4 +16,132 @@ for (i= 0; i < skills.length; i++){
     let skill = document.createElement('li');
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
-}
\ No newline at end of file
+}
+
+
+
+//handle message form submit
+let messageForm = document.getElementsByName("leave_message");
+
+console.log(messageForm[0], "this is the form");
+
+
+
+messageForm[0].addEventListener("submit", function(event){
+    event.preventDefault();
+
+    let theMessageForm = event.target;
+
+    let nameInput = event.target.name.value;
+    let emailInput = event.target.email.value;
+    let messageInput = event.target.message.value;
+
+    //the inputs
+    console.log(nameInput);
+    console.log(emailInput);
+    console.log(messageInput);
+
+    let messageSection = document.getElementById("messages");
+
+    let messageList = messageSection.querySelector("ul");
+
+    let newMessage = document.createElement("li");
+
+    
+    newMessage.innerHTML = `
+    <a href="mailto:${emailInput}">${nameInput}</a>
+    <span>${messageInput}</span>
+    `;
+
+    //adding messages to the list
+    messageList.appendChild(newMessage);
+    
+    
+    //========================THE REMOVE BUTTON===================
+    let removeButton = document.createElement("button");
+    removeButton.type = "button";
+    removeButton.innerHTML = "REMOVE";
+    
+
+    //button removes list item
+    removeButton.addEventListener("click", function(event){
+        event.preventDefault();
+        let entry = removeButton.parentNode;
+        entry.remove();
+
+        //function call to hide or show message list when click remove button
+        hideShow();
+
+    })
+
+    newMessage.appendChild(removeButton);
+
+
+    //======================================================================
+
+
+
+    //========================THE EDIT BUTTON===================
+    // let editButton = document.createElement("button");
+    // editButton.type = "button";
+    // editButton.innerHTML = "EDIT";
+    
+
+    // //button edits list item
+    // editButton.addEventListener("click", function(event){
+    //     event.preventDefault();
+        
+    //     //NOTHING HERE YET. IN PROGRESS. 
+
+    //     console.log("edit button has been clicked");
+
+    //     //function call to hide or show message list when click edit button
+    //     hideShow();
+
+    // })
+
+    // newMessage.appendChild(editButton);
+
+
+    //======================================================================
+
+
+
+
+
+
+    
+
```

Comment:
A little bit of whitespace can help with readability, but you probably never want more than two blank lines.

Reply:
@mccalluc I had all that whitespace because I was working on something I commented out and it was kinda my visual reminder that I was working on something right there and to go back to it later.